src/app/core/store/documents/documents.action.ts

```diff
@@ -31,13 +31,20 @@ export enum DocumentsActionType {
   CREATE_SUCCESS = '[Documents] Create :: Success',
   CREATE_FAILURE = '[Documents] Create :: Failure',
 
-  UPDATE = '[Documents] Update',
   UPDATE_SUCCESS = '[Documents] Update :: Success',
   UPDATE_FAILURE = '[Documents] Update :: Failure',
 
   UPDATE_DATA = '[Documents] Update Data',
   PATCH_DATA = '[Documents] Patch Data',
 
+  ADD_FAVORITE = '[Documents] Add Favorite',
+  ADD_FAVORITE_SUCCESS = '[Documents] Add Favorite :: Success',
+  ADD_FAVORITE_FAILURE = '[Documents] Add Favorite :: Failure',
+
+  REMOVE_FAVORITE = '[Documents] Remove Favorite',
+  REMOVE_FAVORITE_SUCCESS = '[Documents] Remove Favorite :: Success',
+  REMOVE_FAVORITE_FAILURE = '[Documents] Remove Favorite :: Failure',
```

Comment:
Why do we have new actions for these? We should not add a new action for every field in document metadata.

Reply:
It's not metadata. See comment above.